src/matvis/cli.py

```diff
@@ -290,8 +315,10 @@ def hera_profile(hex_num, nside, keep_ants, outriggers, **kwargs):
 
     bls = antpos[np.newaxis, :, :2] - antpos[:, np.newaxis, :2]
     pairs = np.array(get_redundancies(bls.value))
+	matsets = list(get_matrix_sets(bls.value))
```

Comment:
here, you could accept a string value for matsets that specifies a function for how to divide up the sets

Reply:
good idea.  This section of the code can expand to include different functions that we populate into submatrices.py.